README.md

```diff
@@ -67,67 +74,102 @@ This tool uses [glances](https://github.com/nicolargo/glances/).
-        <!--Number of sensors-->
-        <add key="SensorNumber" value="100000" />
+        <add key="ClickhouseDatabase" value="default" />
+ 
+    <!-- General Settings-->
+          <add key="PrintModeEnabled" value="false" />
+          <add key="TestRetries" value="2" />
+          <add key="DaySpan" value="1" />
+        <!-- Could be: DummyDB,  PostgresDB , DatalayertsDB , ClickhouseDB , TimescaleDB , InfluxDB -->
+          <add key="TargetDatabase" value="DummyDB" />
+          <add key="StartTime" value="2022-01-01T00:00:00.00" />
+          <add key="RegularTsScaleMilliseconds" value="1000" /> 
+        <!-- Where to store metrics files: The Programm will split the files in "[...]Read.csv" and "[...]Write.csv" -->
+          <add key="MetricsCSVPath" value="Metrics_Source_Month-Day" />
+          
+    <!-- System Metrics Options -->
+          <add key="GlancesOutput" value="Glances_Source_Month-Day.csv"/>
+          <add key="GlancesUrl" value="http://localhost:61208" />
+          <add key="GlancesDatabasePid" value="1" />
+          <add key="GlancesPeriod" value="1" />
+          <add key="GlancesNIC" value="lo" />
+          <add key="GlancesDisk" value="sda1" />
+          <add key="GlancesStorageFileSystem" value="/" />
+        <!-- Insert multiple dimensionnrs, e.g.  1,6 ,12 ,50, 100, -->
+          <add key="DataDimensionsNrOptions" value="1,6" />  
+
+    <!-- Read Query Options -->
+        <!-- Could be: Agg, All, RangeQueryRawData, RangeQueryRawAllDimsData, RangeQueryRawLimitedData, RangeQueryRawAllDimsLimitedData  RangeQueryAggData, OutOfRangeQuery, DifferenceAggQuery, STDDevQuery -->
+          <add key="QueryType" value="All" />
+          <add key="AggregationIntervalHour" value="1" />
+          <add key="DurationMinutes" value="60" />
+          <add key="SensorsFilter" value="1,2,3,4" /> <!--  or "All" -->
+          <add key="SensorID" value="1" />
+          <add key="MaxValue" value="0.9" />
+          <add key="MinValue" value="0.1" />
+          <add key="FirstSensorID" value="1" />
+          <add key="SecondSensorID" value="2" />
+
+    <!-- Ingestion -->
+        <!-- Could be: regular, irregular -->
+          <add key="IngestionType" value="regular" /> 
+        <!-- Coulde be:  33, 100 , 300  -->
+          <add key="MixedWLPercentageOptions" value="33, 100,300" />
+        <!-- Could be: array, column. Array is not fully implemented in all DBMS. -->
+          <add key="MultiDimensionStorageType" value="column" />
+        <!-- 10, 1000, 5000, 10000 , 50000 -->
+          <add key="BatchSizeOptions" value=" 100 , 1000, 6000 " />
+        <!-- Number of concurrent clients e.g.(1,8,16) must be less than sensors. BatchSizes will be shared out between the clients -->
+          <add key="ClientNumberOptions" value="1 , 8" />
+          <add key="SensorNumber" value="100" />   
 
     </appSettings>
 
 </configuration>
 ```
 
+## Workloads from 2022 benchmark.
```

Comment:
Update the workloads folder and the below table with the one you have used for benchmarking